### PR TITLE
 chore(curriculum): add instructions 07

### PR DESCRIPTION
**Pull request by alevanni**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Related to issue [#43836](https://github.com/freeCodeCamp/freeCodeCamp/issues/43836).
This pull requests updates the remaining descriptions of the certificate #7 (scientific computing with Python). 
For the blocks of code: I used ```bash for the console output, and left ```py where it was, but I was not sure how to format the other blocks of coding, so for now they are unchanged (```).

### CODE REVIEW

File: curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md [hunk 1]
```diff
@@ -8,20 +8,68 @@ dashedName: probability-calculator
...
+
+# --instructions--
+
+Suppose there is a hat containing 5 blue balls, 4 red balls, and 2 green balls. What is the probability that a random draw of 4 balls will contain at least 1 red ball and 2 green balls? While it would be possible to calculate the probability using advanced mathematics, an easier way is to write a program to perform a large number of experiments to estimate an approximate probability.
+
+For this project, you will write a program to determine the approximate probability of drawing certain balls randomly from a hat.
+
+First, create a `Hat` class in `prob_calculator.py`. The class should take a variable number of arguments that specify the number of balls of each color that are in the hat. For example, a class object could be created in any of these ways:
+
+```
```
Comment (nhcarrigan) [inline curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md:27]:
```suggestion
```py
```
This one is python

File: curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md [hunk 2]
```diff
@@ -8,20 +8,68 @@ dashedName: probability-calculator
...
+- `num_balls_drawn`: The number of balls to draw out of the hat in each experiment.
+- `num_experiments`: The number of experiments to perform. (The more experiments performed, the more accurate the approximate probability will be.)
+
+The `experiment` function should return a probability.
+
+For example, let's say that you want to determine the probability of getting at least 2 red balls and 1 green ball when you draw 5 balls from a hat containing 6 black, 4 red, and 3 green. To do this, we perform `N` experiments, count how many times `M` we get at least 2 red balls and 1 green ball, and estimate the probability as `M/N`. Each experiment consists of starting with a hat containing the specified balls, drawing a number of balls, and checking if we got the balls we were attempting to draw.
+
+Here is how you would call the `experiment` function based on the example above with 2000 experiments:
+
+```
```
Comment (nhcarrigan) [inline curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md:50]:
```suggestion
```py
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md [hunk 3]
```diff
@@ -8,20 +8,68 @@ dashedName: probability-calculator
...
+                  num_experiments=2000)
+```
+
+Since this is based on random draws, the probability will be slightly different each time the code is run.
+
+*Hint: Consider using the modules that are already imported at the top of `prob_calculator.py`. Do not initialize random seed within `prob_calculator.py`.*
+
+## Development
+
+Write your code in `prob_calculator.py`. For development, you can use `main.py` to test your code. Click the "run" button and `main.py` will run.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md:64]:
```suggestion
Write your code in `prob_calculator.py`. For development, you can use `main.py` to test your code. Click the "run" button and `main.py` will run.

The boilerplate includes `import` statements for the `copy` and `random` modules. Consider using those in your project.
```
So we can remove the comment from `prob_calculator.py` 🙂

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
One more tiny thing, I think.

The comments within the boilerplate code aren't translatable and our goal is to remove them. For these projects, only one of them has a comment, so I added a suggestion to mirror the verbiage here.

File: curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md (hunk 3 above)
Comment (alevanni) [reply curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md:64]: Thank you for the suggestion, I'll keep the presence of comments in mind for the remaining tasks!

**Review by naomi-lgbt — APPROVED** [review]
Awesome. Thanks for taking care of this!

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Pull request merged**